src/common/util/include/openvino/util/common_util.hpp

```diff
@@ -118,5 +118,8 @@ bool contains(const std::vector<T, A>& vec, const V& v) {
         return x == v;
     });
 }
+
+bool check_all_digits(const std::string& value);
```

Comment:
Why do we need to have this method in public API?
I propose tom move it to developer API if we need to use it in different modules.
But honestly I prefer to have internal header inside `src/core/src/` folder in order to don't export this function to any APIs.

Reply:
Created new header for check_all_digits() in src/core/src/.